troubleshooting/debugging.md

```diff
@@ -860,6 +872,23 @@ If you choose to deploy this, for the S3 upload on shutdown to work, you must:
 
 Make sure to edit the `Resource` section with your bucket name. 
 
+**RUN_ID Environment Variable**
+The RUN_ID environment variable is automatically set to a random value per each run and can be referenced in your Fluent Bit configuration file. It can be used to customize the log destination for each `aws-for-fluent-bit` execution, and help you to find the log destination when looking at a specific Fluent Bit service log. The following log line will appear before Fluent Bit is executed: `RUN_ID is set to $RUN_ID`.
```

Comment:
let's be clear RUN_ID is only a feature for debug

Should we make it a feature for non-debug??

Reply:
Maybe. But we don't have any customer requests for a feature like this. I don't think people would use it. If we did make it a feature for non-debug, we should probably have a name with the AWS_ prefix maybe AWS_FLB_RUN_ID. I don't really think this feature would be widely used.

Reply:
👍 
ok